conf/application.conf

```diff
@@ -53,7 +53,15 @@ play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline'
 
 play.http.requestHandler = "uk.gov.hmrc.play.bootstrap.http.RequestHandler"
 play.http.errorHandler = "handlers.ErrorHandler"
+
 play.http.filters = "filters.Filters"
+
+play.filters.csrf.header.bypassHeaders {
+  X-Requested-With = "*"
+  Csrf-Token = "nocheck"
+}
+play.filters.csrf.token.sign = false
+
```

Comment:
If we do this, we'll need to make sure it's turned back on in every other environment. It's worth talking about whether we can get this to be turned off just for tests rather than globally in the default config